BookingsApi/BookingsApi.IntegrationTests/Api/JusticeUsers/DeleteJusticeUserTests.cs

```diff
@@ -100,7 +100,7 @@ private async Task<JusticeUser> SeedJusticeUser(string username)
             {
                 ContactEmail = username,
                 Username = username,
-                UserRoleId = (int)UserRoleId.Vho,
+                //UserRoles = new []{(int)UserRoleId.Vho},
```

Comment:
should this be commented out? Might be worth assigning the user roles here so that the user is in a realistic/valid state.

in the edit justice user command tests you use a method to seed the user roles, could we use that here too?